Makefile

```diff
@@ -185,6 +185,34 @@ hosts-chrome-user:
 	*)          echo "The operating system $(OS) is not supported"; exit 1 ;; \
 	esac
 
+.PHONY: hosts-edge
+hosts-edge:
+	# THIS DOESN'T WORK
+	@case $(OS) in \
+	Linux)      mkdir -p "/opt/microsoft/msedge/native-messaging-hosts/"; \
```

Comment:
Instead of the comment, let's just have `Linux)` block commented out too, so that the `make` goal exists, but using it would result in getting an error message, hopefully encouraging community to contribute 😜